website/content/docs/api-gateway/install.mdx

```diff
@@ -37,47 +19,46 @@ Ensure that the environment you are deploying Consul API Gateway in meets the re
             name: consul
           connectInject:
             enabled: true
-          apiGateway:
-            enabled: true
-            image: hashicorp/consul-api-gateway:$VERSION
         EOF
       ```
     </CodeBlockConfig>
 
-1. Install Consul API Gateway using the standard Consul Helm chart or Consul K8s CLI specify the custom values file. Available versions of the [Consul Helm chart](https://github.com/hashicorp/consul-k8s/releases) can be found in GitHub releases.
+The Consul Helm chart deploys the API gateway using the configuration specified in the `values.yaml` file. Refer to [Helm Chart Configuration - `apiGateway`](/consul/docs/k8s/helm#apigateway) for information about the Helm chart configuration options.
+
+1. Install Consul API Gateway using the standard Consul Helm chart or Consul K8s CLI specify the custom values file. Refer to the [Consul Helm chart](https://github.com/hashicorp/consul-k8s/releases) in GitHub releases for the available versions.
 
-  <Tabs>
-    <Tab heading="Consul K8s CLI (Mac Only)">
+<Tabs>
+<Tab heading="Consul K8s CLI (Mac Only)">
 
-    ~> **Note:** Refer to the official [Consul K8S CLI documentation](/consul/docs/k8s/k8s-cli) to find additional settings.
+Refer to the official [Consul K8S CLI documentation](/consul/docs/k8s/k8s-cli) to find additional settings.
 
-    ```shell-session
-    $ brew tap hashicorp/tap
-    ```
+```shell-session
+$ brew tap hashicorp/tap
+```
 
-    ```shell-session
-    $ brew install hashicorp/tap/consul-k8s
-    ```
+```shell-session
+$ brew install hashicorp/tap/consul-k8s
+```
 
-    ```shell-session
-    $ consul-k8s install -config-file=values.yaml -set global.image=hashicorp/consul:1.14.3
-    ```
+```shell-session
+$ consul-k8s install -config-file=values.yaml -set global.image=hashicorp/consul:1.16.0
+```
 
-    </Tab>
-    <Tab heading="Helm">
+</Tab>
+<Tab heading="Helm">
 
-    Add the HashiCorp Helm repository.
+Add the HashiCorp Helm repository.
 
-    ```shell-session
-    $ helm repo add hashicorp https://helm.releases.hashicorp.com
-    ```
-    Install Consul with API Gateway on your Kubernetes cluster by specifying the `values.yaml` file.
+```shell-session
+$ helm repo add hashicorp https://helm.releases.hashicorp.com
+```
+Install Consul with API Gateway on your Kubernetes cluster by specifying the `values.yaml` file.
 
-    ```shell-session
-    $ helm install consul hashicorp/consul --version 1.0.2 --values values.yaml --create-namespace --namespace consul
-    ```
-    </Tab>
-  </Tabs>
+```shell-session
+$ helm install consul hashicorp/consul --version 1.0.2 --values values.yaml --create-namespace --namespace consul
```

Comment:
The user will want to install version 1.2 which is what API Gateway is shipping in.